packages/patternfly-4/react-core/src/components/Modal/ModalBoxHeader.js

```diff
@@ -17,7 +17,9 @@ const defaultProps = {
 
 const ModalBoxHeader = ({ children, className, ...props }) => (
   <header {...props} className={css(styles.modalBoxHeader, className)}>
-    <h1 className={css(styles.modalBoxHeaderTitle)}>{children}</h1>
+    <h1 className={css(styles.modalBoxHeaderTitle)} id="modal-header-h1">
```

Comment:
Not use if this is just because it is a WIP or not, but this will cause issues of multiple modals are open. Will need to make this generated or passed in before this merges.

Reply:
There are no plans to merge this one. This is intended to set up examples needed for testing.